From e97249564960289e27ba5b6e996a6ac90e6d9b17 Mon Sep 17 00:00:00 2001
From: Haikel Guemar <hguemar@fedoraproject.org>
Date: Mon, 13 Oct 2014 14:59:39 +0200
Subject: [PATCH] Remove runtime dep on python pbr

---
 trove/__init__.py | 21 +++++++++++++++++++--
 1 file changed, 19 insertions(+), 2 deletions(-)

diff --git a/trove/__init__.py b/trove/__init__.py
index 932f8f7..36b2e63 100644
--- a/trove/__init__.py
+++ b/trove/__init__.py
@@ -24,9 +24,26 @@
 
 __all__ = ['__version__']
 
-import pbr.version
+class VersionInfo(object):
+    release = "REDHATTROVERELEASE"
+    version = "REDHATTROVEVERSION"
 
-version_info = pbr.version.VersionInfo('trove')
+    def version_string(self):
+        return self.version
+
+    def cached_version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+    def canonical_version_string(self):
+        return self.version
+
+    def version_string_with_vcs(self):
+        return self.release
+
+version_info = VersionInfo()
 # We have a circular import problem when we first run python setup.py sdist
 # It's harmless, so deflect it.
 try:
