diff --git a/trove/version.py b/trove/version.py
index 33656d8..504905f 100644
--- a/trove/version.py
+++ b/trove/version.py
@@ -27,19 +27,12 @@ FINAL = False   # This becomes true at Release Candidate time
 
 
 def canonical_version_string():
-    return '.'.join([YEAR, COUNT])
+    return "REDHATVERSION"
 
 
 def version_string():
-    if FINAL:
-        return canonical_version_string()
-    else:
-        return '%s-dev' % (canonical_version_string(),)
-
-
-def vcs_version_string():
-    return "%s:%s" % (version_info['branch_nick'], version_info['revision_id'])
+    return '%s-%s' % (canonical_version_string(),"REDHATRELEASE")
 
 
 def version_string_with_vcs():
-    return "%s-%s" % (canonical_version_string(), vcs_version_string())
+    return version_string()
